package/boot/uboot-mediatek/patches/020-no-mkeficapsule.patch

```diff
@@ -0,0 +1,10 @@
+--- a/tools/Kconfig
++++ b/tools/Kconfig
+@@ -98,7 +98,6 @@ config TOOLS_SHA512
+ 
+ config TOOLS_MKEFICAPSULE
+ 	bool "Build efimkcapsule command"
+-	default y if EFI_LOADER
```

Comment:
Wonder if it can be replaced with `UBOOT_CUSTOMIZE_CONFIG := --disable TOOLS_MKEFICAPSULE` in Makefile.